DeAceroDLabs/marketplace-frontend
# Feature/dmp 424

## What is done

[[DMP-424] Signup Form](https://deacero.atlassian.net/browse/DMP-424)

## How is done

1. Added multiple form component to have multiple forms into one form And to navigate through them.
2. Added new form fields components such as number and dropdown.

## How to test

1. Run the project via `npm start`
4. Got to `/signup``
5. Verify that you see the form as in the mockups.
6. Navigate though the form.
